src/main/java/graphql/relay/Base64.java

```diff
@@ -1,24 +1,25 @@
 package graphql.relay;
 
 import java.nio.charset.Charset;
-import javax.xml.bind.DatatypeConverter;
-import java.io.UnsupportedEncodingException;
-
 
+/**
+ * @deprecated use {@link java.util.Base64} instead.
+ */
+@Deprecated
 public class Base64 {
```

Comment:
I agree in general but the next release is a breaking release. So please delete it. :-) thanks